source/extensions/path/uri_template_lib/uri_template_internal.cc

```diff
@@ -155,24 +157,25 @@ absl::StatusOr<ParsedResult<Operator>> parseOperator(absl::string_view pattern)
   if (absl::StartsWith(pattern, "*")) {
     return ParsedResult<Operator>(Operator::PathGlob, pattern.substr(1));
   }
-  return absl::InvalidArgumentError("Invalid Operator");
+  return absl::InvalidArgumentError(fmt::format("Invalid Operator: \"{}\"", pattern));
```

Comment:
I am trying to see whether we have a test for each added error message.

Do we have test for this error?